internal/cmd/argus/instance/create/create.go

```diff
@@ -0,0 +1,191 @@
+		Example: examples.Build(
+			examples.NewExample(
+				`Create an Argus instance with name "my-instance" and specify plan by name`,
+				"$ stackit argus instance create --name my-instance --plan-name Monitoring-Starter-EU01"),
+			examples.NewExample(
+				`Create an Argus instance with name "my-instance" and specify plan by ID`,
+				"$ stackit argus instance create --name my-instance --plan-id xxx"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			projectLabel, err := projectname.GetProjectName(ctx, cmd)
+			if err != nil {
+				projectLabel = model.ProjectId
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to create an Argus instance for project %q?", projectLabel)
+				err = confirm.PromptForConfirmation(cmd, prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req, err := buildRequest(ctx, model, apiClient)
+			if err != nil {
+				var dsaInvalidPlanError *cliErr.DSAInvalidPlanError
+				if !errors.As(err, &dsaInvalidPlanError) {
+					return fmt.Errorf("build Argus instance creation request: %w", err)
+				}
+				return err
+			}
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("create Argus instance: %w", err)
+			}
+			instanceId := *resp.InstanceId
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(cmd)
+				s.Start("Creating instance")
+				_, err = wait.CreateInstanceWaitHandler(ctx, apiClient, instanceId, model.ProjectId).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for Argus instance creation: %w", err)
+				}
+				s.Stop()
+			}
+
+			operationState := "Created"
+			if model.Async {
+				operationState = "Triggered creation of"
+			}
+			cmd.Printf("%s instance for project %q. Instance ID: %s\n", operationState, projectLabel, instanceId)
+			return nil
+		},
+	}
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP(instanceNameFlag, "n", "", "Instance name")
+	cmd.Flags().Var(flags.UUIDFlag(), planIdFlag, "Plan ID")
+	cmd.Flags().String(planNameFlag, "", "Plan name")
+
+	err := flags.MarkFlagsRequired(cmd, instanceNameFlag)
+	cobra.CheckErr(err)
+}
+
+func parseInput(cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &cliErr.ProjectIdError{}
+	}
+
+	planId := flags.FlagToStringPointer(cmd, planIdFlag)
+	planName := flags.FlagToStringValue(cmd, planNameFlag)
+
+	if planId == nil && (planName == "") {
+		return nil, &cliErr.DSAInputPlanError{
+			Cmd: cmd,
+		}
+	}
+	if planId != nil && (planName != "") {
+		return nil, &cliErr.DSAInputPlanError{
+			Cmd: cmd,
+		}
+	}
```

Comment:
In here we should create a new error type `ArgusInputPlanError`

`DSAInputPlanError` is used for DSA services (`redis`, `mariadb`, `logme`, `rabbitmq`, and `opensearch`) where the plan is given as a combination of plan name and version. You can still take this implementation as reference